src/boot/fp.c

```diff
@@ -0,0 +1,41 @@
+#include "global.h"
```

Comment:
If we're not sure whether this is handwritten, should we err on the side of not giving it a C file for now?

Reply:
I removed the c file, is there any changes I need to make to the progress script or any other tools?

Reply:
I think only the progress script cares about C vs not C, and only when it bins up the asm and src totals, not when it actually computes the decompilable bytes (that hasn't changed yet, although it will when we're certain about handwritten files). The progress script says this for the previous commit (merging my PR):
```
src:     568092 /  4756800 total bytes decompiled      11.9427%
    boot:         46488 /    92320 bytes decompiled      50.3553%
    code:        159384 /  1068896 bytes decompiled      14.9111%
    overlays:    362220 /  3595584 bytes decompiled      10.0740%
```
so it'll be fine if those numbers are the same.